core/designsystem/src/main/java/com/dev/innerview/core/designsystem/component/TextField.kt

```diff
@@ -0,0 +1,104 @@
+package com.dev.innerview.core.designsystem.component
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.dev.innerview.core.designsystem.theme.InnerViewTheme
+import com.dev.innerview.core.designsystem.theme.Paddings
+
+@Composable
+fun InnerViewDialogTextField(
+    value: () -> String,
+    onValueChange: (String) -> Unit,
+    singleLine: Boolean = true,
+    enabled: Boolean = true,
+    placeholderText: String = "",
+    labelText: String = "",
+    supportingText: String = "",
+) {
+    BasicTextField(
+        modifier = Modifier
```

Comment:
🚫 **[ktlint]** <standard:multiline-expression-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A multiline expression should start on a new line
<!-- __reviewdog__:ChBiZmU1MjNlOTZmNDE4ZjYzEgZrdGxpbnQ= -->